integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/e2e/freeipa/FreeIpaUpgradeTests.java

```diff
@@ -194,6 +197,21 @@ private void createBindUser(TestContext testContext, com.sequenceiq.freeipa.api.
         }
     }
 
+    private void generateHostKeyTab(com.sequenceiq.freeipa.api.client.FreeIpaClient ipaClient, String environmentCrn) {
+        try {
+            HostKeytabRequest hostKeytabRequest = new HostKeytabRequest();
+            hostKeytabRequest.setEnvironmentCrn(environmentCrn);
+            hostKeytabRequest.setServerHostName("test.local");
+            hostKeytabRequest.setRoleRequest(new RoleRequest());
+            hostKeytabRequest.setDoNotRecreateKeytab(Boolean.TRUE);
```

Comment:
in this case the code won't call the FreeIPA itself if the keytab already exists and cached, so the availability won't be really tested

Reply:
Done.